test/gleam/list_test.gleam

```diff
@@ -846,11 +846,44 @@ pub fn key_set_test() {
 }
 
 pub fn each_test() {
-  list.each([1, 1, 1], fn(x) { let assert 1 = x })
+  list.each(
+    [1, 1, 1],
+    fn(x) {
+      let assert 1 = x
+    },
+  )
   |> should.equal(Nil)
 
   // TCO test
-  list.each(list.repeat(1, recursion_test_cycles), fn(x) { let assert 1 = x })
+  list.each(
+    list.repeat(1, recursion_test_cycles),
+    fn(x) {
+      let assert 1 = x
+    },
+  )
+}
+
+pub fn try_each_test() {
+  list.try_each(
+    over: [1, 1, 1],
+    with: fn(x) {
+      let assert 1 = x
```

Comment:
I am not sure I understand this line, but it could very well be me.

Reply:
I saw the `let assert` in the tests for `list.each` and thought it would be good. However, now I've changed it to `should.equal`, is it better?

Reply:
I like it, but I am not louis :D

Also I am confused that `let assert 1 = x` works, does it bind `True` to `x`? or 1 to `x`?
It should panic if x is not 1 and fail the test.

Thus I think should.equal is read and understood easier.

Reply:
Yeah it puzzled me as well! I definitely prefer the `should.equal`